index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vangelis</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="header">
+      <!-- Navigation Bar w/ Welcome message and Hamburger -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li>
+            <a href="#"><img src="/images/Union.png" alt="toggle-switch" /></a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <!-- End of Navigation Bar -->
+
+    <!-- Main body w/ hello I am Vangelis and some more text -->
+    <section class="main-body">
+      <!-- <img src="/images/header-illsutration-mobile.png" /> -->
+      <!-- 1 -->
+      <div class="title">
+        <h1>Hey there. I'm<br />Vangelis<br /></h1>
+      </div>
+      <!-- 2 -->
+      <div class="profession">
+        <h1>I'm a software developer</h1>
+      </div>
+      <!-- 3 -->
+      <div class="description">
+        <p>
+          I can help you build a product,<br />
+          feature or website. Look through<br />
+          some of my work and experience!<br />
+          If you like what you see and have<br />
+          a project you need coded, <br />
+          don't hesitate to contact me.
+        </p>
+      </div>
+      <!-- 4 Start of with social media -->
+      <div class="social-icon-container">
+        <ul>
+          <li class="social">
+            <a href="#"><img src="/images/twitter.svg" alt="twitter-icon" /></a>
+          </li>
+          <li class="social">
+            <a href="#"
+              ><img src="/images/linkedin.svg" alt="linkedin-icon"
+            /></a>
+          </li>
+          <li class="social">
+            <a href="#"><img src="/images/medium.svg" alt="medium-icon" /></a>
+          </li>
+          <li class="social">
+            <a href="#"><img src="/images/github.svg" alt="github-icon" /></a>
+          </li>
+          <li class="social">
+            <a href="#"
+              ><img src="/images/angellist.svg" alt="angellist-icon"
```

Comment:
- [ ] [ OPTIONAL ] To ensure that your images work perfectly across all browsers, kindly avoid adding a `/` at the beginning of the image url. Consider using `"images/myimage.png"` instead of  `"/images/myimage.png"`